src/test/java/kitchenpos/application/OrderServiceTest.java

```diff
@@ -0,0 +1,130 @@
+package kitchenpos.application;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.util.ArrayList;
+import java.util.List;
+import kitchenpos.domain.Order;
+import kitchenpos.domain.OrderLineItem;
+import kitchenpos.domain.OrderStatus;
+import kitchenpos.domain.OrderTable;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.transaction.annotation.Transactional;
+
+@SpringBootTest
+@Transactional
+class OrderServiceTest {
+
+    @Autowired
+    private TableService tableService;
+
+    @Autowired
+    private OrderService orderService;
+
+    private OrderTable orderTable;
+    private Order order;
+
+    @BeforeEach
+    void setUp() {
+        orderTable = tableService.create(new OrderTable());
+        order = new Order();
+        order.setOrderTableId(orderTable.getId());
+    }
+
+    @Test
+    @DisplayName("주문을 생성한다")
+    void create() {
+        final List<OrderLineItem> orderLineItems = new ArrayList<>();
+        final OrderLineItem orderLineItem = generateOrderLineItem(1L, 1);
+        orderLineItems.add(orderLineItem);
+        order.setOrderLineItems(orderLineItems);
+
+        final Order createOrder = orderService.create(order);
+
+        assertThat(createOrder.getOrderTableId())
+                .isEqualTo(orderTable.getId());
+    }
+
+    @Test
+    @DisplayName("메뉴가 없는 주문을 생성하면 예외를 반환한다")
+    void create_notHaveMenuException() {
+        final List<OrderLineItem> orderLineItems = new ArrayList<>();
+        order.setOrderLineItems(orderLineItems);
+
+        assertThatThrownBy(() -> orderService.create(order))
+                .isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @Test
+    @DisplayName("존재하지 않는 메뉴로 주문을 생성하면 예외를 반환한다")
+    void create_notExistMenuException() {
+        final List<OrderLineItem> orderLineItems = new ArrayList<>();
+        final OrderLineItem orderLineItem = generateOrderLineItem(999999999L, 1);
+        orderLineItems.add(orderLineItem);
+        order.setOrderLineItems(orderLineItems);
+
+        assertThatThrownBy(() -> orderService.create(order))
+                .isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @Test
+    @DisplayName("주문 전체를 조회한다")
+    void list() {
+        final List<OrderLineItem> orderLineItems = new ArrayList<>();
+        final OrderLineItem orderLineItem = generateOrderLineItem(1L, 1);
+        orderLineItems.add(orderLineItem);
+        order.setOrderLineItems(orderLineItems);
+
+        orderService.create(order);
+
+        assertThat(orderService.list())
+                .hasSizeGreaterThan(0);
```

Comment:
```suggestion
        assertThat(orderService.list())
                .isNotEmpty();
```
이런식으로 하는게 더 직관적이지 않을까 싶습니다!